layouts/partials/article/root.html

```diff
@@ -11,11 +11,13 @@
 
 {{/* Unique id for this article */}}
 {{ $articleId := .Params.id | default .File.Path }}
+{{ $noContent := (eq (len .Content) 0) }}
+{{ $disableArticleDelete := (eq $type "parent") }}
 
 {{/* Unique id for file, used by scroll-spy and menu toggle  */}}
 {{ $uid := .File.UniqueID }}
 
-<div class="article {{ $type }}" data-roles="{{ $roles }}" id="{{ $uid }}" data-link="{{ $articleLink }}" permalink="{{.Page.RelPermalink}}" >
+<div class="article {{ $type }}" data-roles="{{ $roles }}" id="{{ $uid }}" data-link="{{ $articleLink }}" permalink="{{.Page.RelPermalink}}"  data-disable-article-bar="{{ $noContent }}" data-disable-article-options="false" data-disable-article-delete="{{ $disableArticleDelete }}" >
```

Comment:
We should rather name this something like `data-content-length`.  Then we can use this attribute to do various things on the JS side.  Otherwise, we need to come add/remove attributes here whenever we want to do something else with empty content articles.